build-tests/api-extractor-test-01/dist/api-extractor-test-01-beta.d.ts

```diff
@@ -1,325 +0,0 @@
-/**
```

Comment:
Why is this part of the PR?

Reply:
These are untracked in .git because they are in the dist folder, this is likely a branch switching artifact that is getting cleaned up appropriately. It technically shouldn't be in main either.

Reply:
Removed thanks to change from @iclanton upstream